conda_smithy/configure_feedstock.py

```diff
@@ -242,11 +247,15 @@ def render_travis(jinja_env, forge_config, forge_dir):
     meta = forge_config['package']
     with fudge_subdir('osx-64', build_config=meta_config(meta)):
         meta.parse_again()
-        matrix = compute_build_matrix(
-            meta,
-            forge_config.get('matrix'),
-            forge_config.get('channels', {}).get('sources', tuple())
-        )
+        if meta.noarch:
+            # do not build noarch, including noarch: python, packages on Travis CI.
+            matrix = ()
```

Comment:
This is typically a `list`. I don't think there is any harm with it being a `tuple`, but we should be consistent here.